connector_woocommerce/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```

Comment:
Hi,
Please follow: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Document title lines must be as long as the title